blacken_docs.py

```diff
@@ -20,9 +20,10 @@
     re.DOTALL | re.MULTILINE,
 )
 PY_LANGS = '(python|py|sage|python3|py3|numpy)'
+BLOCK_TYPES = '(code|code-block|sourcecode|ipython)'
 RST_RE = re.compile(
     rf'(?P<before>'
-    rf'^(?P<indent> *)\.\. (code|code-block|sourcecode|ipython):: {PY_LANGS}\n'
+    rf'^(?P<indent> *)\.\. (jupyter-execute::|(BLOCK_TYPES):: {PY_LANGS})\n'
```

Comment:
this needs to be curly braces so it substitutes the string above

Reply:
Oops, sorry 😅